Extensions/disable_gifs.js

```diff
@@ -50,7 +50,7 @@ XKit.extensions.disable_gifs = new Object({
 					z-index: 2;
 				}
 				${XKit.css_map.keyToCss("baseContainer")} {
-					z-index: 3;
+					z-index: 80;
```

Comment:
while trying to make my case that Shorten Posts should z-index its UI lower, I realised that the real issue is that extensions should agree on what to z-index the `baseContainer` to, in order to prevent any weird interactions with each other.

given that Shorten Posts sets it to 99, let's set it to 99 here too, so that Disable GIFs can't lower the value when Shorten Posts has already set it.

```suggestion
					z-index: 99;
```

Reply:
Agreed they should agree, for sure. I think we should use a lower value though in case we ever need to insert something between baseContainer and [header/peepr/whatever], which are at 100. Someone (at Tumblr) not leaving a gap is how we wound up modifying baseContainer in the first place, right? :D

Reply:
sure, but we can fix that when we get to it and not when it's a point of conflict/contention between two PRs. i don't want to request changes on the Shorten Posts PR just to address this specifically, even if it might need to be changed later - if it does, it can be done then, in one PR.

Reply:
Sounds good to me!